include/sound/sof/dai-amd.h

```diff
@@ -17,6 +17,7 @@ struct sof_ipc_dai_acp_params {
 
 	uint32_t fsync_rate;    /* FSYNC frequency in Hz */
 	uint32_t tdm_slots;
+	uint32_t tdm_mode;
```

Comment:
Yes this will be backward compatible, 

As we are doing memset  the structure to zero and then assign the values if present from topology. 

In firmware they will only assign/use the members present.

Onemore thing is as this structure are in union we got benefit of the size concern.